INSTALL

```diff
@@ -1,8 +1,32 @@
-INSTALL file for Workflow
+INSTALL documentation for Workflow
 
-$Id$
+The recommended way of installing the Workflow is from CPAN.
 
-perl Makefile.PL  or   perl Build.PL
-make                   ./Build
-make test              ./Build test
-make install           ./Build install
+# Using CPAN
+$ cpan module_name
```

Comment:
Should we not make this "Workflow specific" too?

Reply:
of course, brain fart on my behalf